turbopy/diagnostics.py

```diff
@@ -109,6 +110,23 @@ def _write_buffer(self):
             np.savetxt(f, self._buffer, delimiter=",")
 
 
+
+class NetCDFOutputUtility(OutputUtility):
```

Comment:
Does this class need to override the `diagnose` and `finalize` methods from the `OutputUtility` base class?